index.html

```diff
@@ -1203,7 +1203,6 @@ <h4>
     <pre class="example">
       const reader = new NDEFReader();
       await reader.scan({
-        recordType: "mime",
```

Comment:
`recordType` not needed? are we counting on the implementation inferring it should be "mime" type?

Reply:
This is scanning, not writing.